HelpSource/scdoc.js

```diff
@@ -198,8 +198,13 @@ function fixTOC() {
 // Set up a QWebChannel for communicating with C++ IDE objects. The main app publishes a handle
 // to IDE functionality at "IDE" which is made globally available here after the page and
 // WebSocket have loaded.
-function setUpWebChannel() {
-    var baseUrl = "ws://localhost:12344";
+
+// This function is called by the IDE, and will not be called otherwise.
+function setUpWebChannel(port) {
+    if (typeof QWebChannel === "undefined") {
```

Comment:
is it even possible for this to evaluate to false when it's called from the IDE code?